src/ucp/core/ucp_context.c

```diff
@@ -354,6 +354,11 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "Use two stage pipeline rendezvous protocol for intra-node GPU to GPU transfers",
    ucs_offsetof(ucp_context_config_t, rndv_shm_ppln_enable), UCS_CONFIG_TYPE_BOOL},
 
+  {"RNDV_PIPELINE_ERRH_ENABLE", "n",
```

Comment:
An idea came to my mind that maybe it worth to introduce more common variable that allows using all the protocols which don't have implemented abort even if error handling is enabled. That control will allow customers to use all the protocols if they are OK that any abort call will cause hard error. Also it can be useful for testing and debugging from my point of view.

Reply:
I think this idea might be useful, but maybe it's out of scope of this PR. Because in this particular case we plan to implement proper support for error handling and therefore this env var will be removed in the subsequent commit. So for these protocols it's not gonna be used anymore, and for the other ones - it can be addressed separately with clear scope & testing